### PR TITLE
[FIXED JENKINS-30483] Pull request is not triggered if CSR is enabled

### PR DESCRIPTION
**Pull request by fbelzunc**
Pull request is not triggered if Prevent Cross Site Request Forgery exploits is enabled

@reviewbybees

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/ghprb/GhprbCrumbExclusion.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            throws IOException, ServletException {
+        final String pathInfo = req.getPathInfo();
+        if (pathInfo != null && pathInfo.startsWith(getExclusionPath())) {
+            chain.doFilter(req, resp);
+            return true;
+        }
+        return false;
+    }
+
+    public String getExclusionPath() {
```
Comment (oleg-nenashev) [inline src/main/java/org/jenkinsci/plugins/ghprb/GhprbCrumbExclusion.java]: May be converted to static/private BTW

**Comment by oleg-nenashev** [comment]
Seems something is wrong with the PR builder

File: src/main/java/org/jenkinsci/plugins/ghprb/GhprbCrumbExclusion.java (hunk 1 above)
Comment (rsandell) [inline src/main/java/org/jenkinsci/plugins/ghprb/GhprbCrumbExclusion.java]: or even ```private static final String EXCLUSION_PATH = "/" + GhprbRootAction.URL + "/"```

**Comment by tfennelly** [comment]
Strange ... the class the PR builder is complaining about doesn't look to be something adding this PR?

**Comment by jglick** [comment]
:bee: (but investigate test failures)

**Comment by DavidTanner** [comment]
Sometimes the tests fail if the PR builder gets multiple jobs.  I ran the tests locally and didn't receive any errors.  @fbelzunc @jglick if you guys are okay with it, I can just merge it anyways.

**Comment by fbelzunc** [comment]
@DavidTanner Yes, please. Merge it.

**Comment by jglick** [comment]
> Sometimes the tests fail if the PR builder gets multiple jobs.

If you cannot find the reason for the failure, you can add `@RandomlyFails` as a workaround.

**Comment by DavidTanner** [comment]
@jglick how do I do that?

**Comment by jglick** [comment]
> how do I do that?

[example](https://github.com/jenkinsci/maven-plugin/blob/f32d1924477e9a5eb34b013d6a9815f4a53d0e0b/src/test/java/hudson/maven/reporters/SurefireArchiverUnitTest.java#L105)

**Pull request merged**